dts/bindings/radio_coex/sdc-radio-coex-three-wire.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+description: |
+    This is a representation of an external radio coexistence setup that has a
+    three-pin control interface (REQUEST, GRANT, PRIORITY/DIRECTION). This
+    extends that interface by adding additional parameters needed by the
+    Softdevice Controller's implementation.
+
+compatible: "sdc-radio-coex-three-wire"
+
+include: generic-radio-coex-three-wire.yaml
+
+properties:
+    is-rx-active-level:
+        type: int
+        required: true
```

Comment:
Can you explain why you choose a required int instead of a boolean?

If you really need an int, I think it would be an improvement to use an `enum` to restrict the allowed values to the two documented ones.

Reply:
I was a bit unsure about booleans, from my initial reading of the [DT manual](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/dts/intro.html#writing-property-values), I thought the syntax was a bit weird with the ``/delete-property/`` and all. I'll change it to a boolean since it's one indeed and our example has it to 0, so it'll demonstrate the way to set it to 0.